src/components/Layout/FormPageLayout.tsx

```diff
@@ -28,17 +29,18 @@ type FormPageLayoutProps<F> = {
   submitAction?: ActionButtonProps;
   cancelAction?: ActionButtonProps;
   tertiaryAction?: ActionButtonProps;
+  rightSideBarContent?: SidebarContentProps[];
```

Comment:
Thought: maybe just call this `rightSideBar`, i.e. the `...Content` is kind of implied by being within the props API.

Reply:
sounds good, I will update this